### PR TITLE
IRO-1657 - Improved build configuration.

### PR DESCRIPTION
**Pull request by ndemidovich-sc**
Improved `tsconfig.json`.
Checked correct work in another projects via `yarn link` and `yarn pack`.

> **NOTE!** If you use this package via `yarn link` in another project, then need solve the problem with React package duplication.
For example if you use `webpack`, then add following to config:
```
{
  resolve: {
    alias: { react: require.resolve("react") }
  },
}
```

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import IronFishUIProvider from 'components/IronFishUIProvider'
-import ColorModeSwitcher from 'components/ColorModeSwitcher'
+import IronFishUIProvider from '../components/IronFishUIProvider'
```
Comment (brekk) [inline .storybook/preview.js:1]: I'd like to keep absolute links.

**Review by brekk — CHANGES_REQUESTED** [review]
Generally these changes are good, but I'd like to be able to keep absolute pathing.

File: .storybook/preview.js (hunk 1 above)
Comment (ndemidovich-sc) [reply .storybook/preview.js]: By default, Typescript does not resolve path aliases when compiling code.
This is solved by adding an additional package and a post-compilation step.

**Review by brekk — APPROVED** [review]
If possible please add the `bureaucracy` step to the `nps care` instructions so we can avoid non-static dependencies by default.

Marking this as approved so we can get it merged in.

File: package.json [hunk 2]
```diff
@@ -62,6 +64,7 @@
     "react-dom": "17.0.2",
     "snang": "0.2.7",
     "ts-jest": "27.1.3",
+    "tsc-alias": "^1.6.5",
```
[the remark this answers is not in the document]
Comment (ndemidovich-sc) [reply package.json]: Fixed.

**Comment by ndemidovich-sc** [comment]
> If possible please add the `bureaucracy` step to the `nps care` instructions so we can avoid non-static dependencies by default.

Step `bureaucracy` is added to `nps care` command. Also removed step `build` from it, because `build` and `test` can took long time. They must be called in CI/CD

**Pull request merged**